<?xml version="1.0"?>
<launch>
    <include file="$(find spinnaker_sdk_camera_driver)/launch/acquisition_node.launch">
      <!--<arg name="nodelet_manager_name" value="bluerov_nodelet_manager"/> -->
      <arg name="node_name" value="front_camera_node"/>
      <arg name="config_file" value="$(find droplet_underwater_assembly)/param/flir_senko_land_calibration.yaml"/>
      <!--<arg name="start_nodelet_manager" value="true"/>-->
      <arg name="color" value="false"/>
      <arg name="exposure_time" value="10000"/>
    </include>

    <include file="$(find  mini_ahrs_ros)/launch/mini_ahrs_standalone.launch"/>

    <node name="gripper_handler" pkg="localization_informed_planning_sim" type="gripper_handler_node.py" required="true" output="screen">

    <node name="localization" pkg="localization_informed_planning_sim" type="breadcrumb_localization_node.py" required="true" output="screen">
	    <param name="fisheye_mode" type="bool" value="true"/>
	    <param name="image_topic" type="string" value="/camera_array/cam1/image_raw"/>
	    <param name="camera_info_topic" type="string" value="/camera_array/cam1/camera_info"/>
    </node>
</launch>
